charts/fluent-bit/values.yaml

```diff
@@ -93,6 +93,9 @@ extraVolumes: []
 
 extraVolumeMounts: []
 
+rollingUpdate:
```

Comment:
WDYT about putting `updateStrategy` here and porting the entire yaml if set? by default have it unset, with a `with`/`toYaml`/`end`  approach in the template?

Reply:
💯 
all in for it. One moment 😬